src/uct/ib/dc/dc_mlx5.h

```diff
@@ -67,12 +68,13 @@ typedef struct uct_dc_mlx5_iface  uct_dc_mlx5_iface_t;
 
 
 typedef enum {
-    UCT_DC_MLX5_IFACE_ADDR_HW_TM      = UCS_BIT(0),
-    UCT_DC_MLX5_IFACE_ADDR_DC_V1      = UCS_BIT(1),
-    UCT_DC_MLX5_IFACE_ADDR_DC_V2      = UCS_BIT(2),
-    UCT_DC_MLX5_IFACE_ADDR_FLUSH_RKEY = UCS_BIT(3),
-    UCT_DC_MLX5_IFACE_ADDR_DC_VERS    = UCT_DC_MLX5_IFACE_ADDR_DC_V1 |
-                                        UCT_DC_MLX5_IFACE_ADDR_DC_V2
+    UCT_DC_MLX5_IFACE_ADDR_HW_TM                = UCS_BIT(0),
+    UCT_DC_MLX5_IFACE_ADDR_DC_V1                = UCS_BIT(1),
+    UCT_DC_MLX5_IFACE_ADDR_DC_V2                = UCS_BIT(2),
+    UCT_DC_MLX5_IFACE_ADDR_FLUSH_RKEY           = UCS_BIT(3),
+    UCT_DC_MLX5_IFACE_ADDR_MAX_RD_ATOMIC_LEGACY = UCS_BIT(4),
```

Comment:
maybe explicitly state what is the value?
e.g UCT_DC_MLX5_IFACE_ADDR_MAX_RD_ATOMIC_16
consider it can be changed by user config..

Reply:
renaming is fine, but  what do you mean by user config?

Reply:
> renaming is fine, but what do you mean by user config?

That since it can be changed by user config, it can still be "legacy" (e.g old device) but with a value different that 16, or new device with a value of 16

Reply:
> > renaming is fine, but what do you mean by user config?
> 
> That since it can be changed by user config, it can still be "legacy" (e.g old device) but with a value different that 16, or new device with a value of 16

So we should rename "legacy" to 16

Reply:
imo legacy is more self-explanatory, but can change to 16 instead.

Reply:
fixed in place because patch is small